### ISSUE TITLE
Not working on  Nokia 7 oreo

### ISSUE THREAD

**Issue by 4512369781** [F-i-f/ResolverActivityTweaks#1]
Checked and restarted, but it does not work, there is no error log in the xposed installer.
nokia 7 Stock rom C1N-224G-0-00CN-B01
xposed v90.2-beta3(Systemless)
magisk v18.1
![screenshot_2019-02-26-21-29-44](https://user-images.githubusercontent.com/30334394/53417108-cc6b1100-3a0f-11e9-9005-0117b3dd935e.png)
![screenshot_2019-02-26-21-30-11](https://user-images.githubusercontent.com/30334394/53417110-cd03a780-3a0f-11e9-9684-064efa9acaf3.png)

**Comment by F-i-f** [comment]
Thank you for your bug report.
The lack of Xposed log will make this issue hard to troubleshoot.
Nevertheless, there's a new release out, 3.0 (in beta on the Xposed module repository) that may well fix this issue.
Would you mind trying it out and reporting back?

**Comment by 4512369781** [comment]
> Thank you for your bug report.
> The lack of Xposed log will make this issue hard to troubleshoot.
> Nevertheless, there's a new release out, 3.0 (in beta on the Xposed module repository) that may well fix this issue.
> Would you mind trying it out and reporting back?

Thanks for your hard work, v3.0 works very well! :)
However, there is a small problem. The program that has been selected only once will not be ranked first, and it will take multiple clicks to increase the ranking. 
Could you modify it?

**Comment by F-i-f** [comment]
No, that's further deep down in the Android code.
That's out of scope for this module.
Thanks.
